app/models/patient.rb

```diff
@@ -82,6 +82,10 @@ class Patient < ApplicationRecord
       .merge(PatientPhoneNumber.phone_type_mobile)
   }
 
+  scope :syncable_to_region, ->(region) {
+    with_discarded.where(registration_facility: region.facilities)
```

Comment:
✨ I really dig the incremental improvements we are getting with the whole region abstraction. Eventually this `region` can just be a Block region, and things will work nicely.

Reply:
Exactly! That's what I had in mind with this change as well.

![image](https://user-images.githubusercontent.com/4241399/95293137-7f5c4b80-0890-11eb-8eca-a5c1f2d726b9.png)